packages/package_info_plus/package_info_plus_web/lib/package_info_plus_web.dart

```diff
@@ -43,6 +43,7 @@ class PackageInfoPlugin extends PackageInfoPlatform {
       buildNumber: versionMap['build_number'] ?? '',
       packageName: versionMap['package_name'] ?? '',
       buildSignature: '',
+      installerStore: null,
```

Comment:
I'm wondering whether it makes sense to return the current URL here

Reply:
I think null is fine

Reply:
Agree that `null` is fine here.

Reply:
Gotcha, `null` it is.